__DEFINES/setup.dm

```diff
@@ -502,6 +503,7 @@ var/global/list/NOIRMATRIX = list(0.33,0.33,0.33,0,\
 #define NERVOUS			16
 #define ASTHMA		32
 #define LACTOSE		64
+#define LISP 		128
```

Comment:
You don't seem to be using this anywhere. Why is this here?

Reply:
I was just copying the format of asthma honestly, if it's unnecessary I'll cut it out.

Reply:
From my limited understanding of code I'm pretty sure this is used for Genetics

Reply:
delet